README.md

```diff
@@ -430,7 +430,7 @@ The query API allows accessing pushed metrics and build and runtime information.
 
 * For example :
 
-        curl -X GET http://pushgateway.example.org:9091/api/v1/status
+        curl -X GET http://pushgateway.example.org:9091/api/v1/status|jq
```

Comment:
Good idea, but could you separate the pipe symbol with spaces around it to make this more readable? I.e. `curl -X GET http://pushgateway.example.org:9091/api/v1/status | jq`

Reply:
I have modify and commit again